watertap/data/techno_economic/tests/test_unit_parameter_files.py

```diff
@@ -55,7 +55,7 @@ def test_unit_parameter_files(tech):
                     "static_mixer",
                     "co2_addition",
                     "sw_onshore_intake",
-                    ]
+                    "landfill_zld"]
```

Comment:
```suggestion
                    "landfill_zld",
                    ]
```

Leaving the trailing comma helps avoid merge conflicts.